testsuite/features/step_definitions/api_common.rb

```diff
@@ -178,6 +178,36 @@
   assert_equal(false, ret.key?(attr))
 end
 
+Then(/^channel "([^"]*)" should (not )?be enabled on "([^"]*)"$/) do |channel, disabled, host|
+  node = get_target(host)
+  system_id = get_system_id(node)
```

Comment:
get_target is an ssh (twopence) node

don't we have an API function to get the system id, while we're at using the API ?

Reply:
I don't really understood the question, but keep in mind this method is defined as:
```
def get_system_id(node)
  $api_test.system.search_by_name(node.full_hostname).first['id']
end
```

Reply:
Good point. I'll look into that and see if we can refactor.